### PR TITLE
move the util functions to its own module

### PR DESCRIPTION
**Pull request by chengxiangdong**
**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

File: pkg/cloudprovider/huaweicloud/elb.go [hunk 1]
```diff
@@ -942,3 +944,110 @@ func (elb *ELBCloud) getSessionAffinityOptions(service *v1.Service) (map[string]
...
+			service, err = kubeClient.Services(service.Namespace).Get(context.TODO(), service.Name, metav1.GetOptions{})
+			if err != nil {
+				klog.Warningf("Get service(%s/%s) error: %v", service.Namespace, service.Name, err)
+				continue
+			}
+		}
+	}
+}
+
+// if async job is success, need to init mark again
```
Comment (niuzhenguo) [inline pkg/cloudprovider/huaweicloud/elb.go:1009]: s/is success/succeed

Comment (chengxiangdong) [reply pkg/cloudprovider/huaweicloud/elb.go]: fixed

**Comment by chengxiangdong** [comment]
@niuzhenguo 
All changes have been pushed, pls review.

**Pull request merged**